src/projects/data/nightly_strategy.md

```diff
@@ -4,5 +4,6 @@ tags: data
 link_href: https://drive.google.com/file/d/1OdYMe0eRVSR79DCtTiGXHCAcaz91Jdyj/view?usp=sharing
 source_href: https://github.com/tombohub/stock-charts/blob/master/src/scripts/nightly_backtest.py
 tech: Python, Pandas, Pinkfish
-content: Backtesting of 5000+ stocks for overnight strategy. Creating plots comparing it to benchmarks
 ---
+
+Backtesting of 5000+ stocks for overnight strategy. Creating plots comparing it to benchmarks
```

Comment:
Rest of these diffs are just converting the frontmatter `content` props into page `templateContent` to make rich formatting easier.